### ISSUE TITLE
replace command having seziures

### ISSUE THREAD

**Issue by deadbushmc** [CzechPMDevs/BuilderTools#198]
**Describe the bug**
so i used the command `//replace stone cobblestone` it works
but when I use `//replace 1:0 236:0, 236:2, 236:3, 236:4, 236:5`  it says 
![image](https://user-images.githubusercontent.com/66917109/154419668-67fad4ba-9cf3-4eec-800c-5722e38536f1.png)

`//replace stone 236:0, 236:2, 236:3, 236:4, 236:5` it says 
![image](https://user-images.githubusercontent.com/66917109/154419634-55e30d2d-8a80-4761-b69a-fa1505c785ae.png)

**To Reproduce**

**Expected behavior**

**Screenshots**

**System information (please complete the following information):**
 - OS: Windows
 - php version [8.0]
 - BuilderTools Version Latest Release

**Additional context**

**Comment by VixikHD** [comment]
Try latest dev build

**Comment by SigitGamers** [comment]
Yes this has Latest, this command *//replace slime* as replace, and use //replace 5:0 5:1 not replaced

**Comment by SigitGamers** [comment]
Command fill not worked, nothing block has fill

**Comment by VixikHD** [comment]
> Command fill not worked, nothing block has fill

???

**Comment by SigitGamers** [comment]
> > Command fill not worked, nothing block has fill
> 
> ???

Can't using item id:(, this work only use item name //fill slime has work and item id not worked to use on cmd //fill 5 has send error message
![IMG-20220214-WA0024](https://user-images.githubusercontent.com/74124100/154867281-7b200008-22c0-4d0e-b33d-a7ef3493526e.jpg)

**Comment by TukangM** [comment]
I'm sure it's a world format issue. if you just reinstalled pocketmine by default the map format becomes leveldb (vanilla bedrock maps)

> Try latest dev build



> > > Command fill not worked, nothing block has fill
> > 
> > 
> > ???
> 
> Can't using item id:(, this work only use item name //fill slime has work and item id not worked to use on cmd //fill 5 has send error message ![IMG-20220214-WA0024](https://user-images.githubusercontent.com/74124100/154867281-7b200008-22c0-4d0e-b33d-a7ef3493526e.jpg)

**Comment by TukangM** [comment]
pocketmine.yml:
level-settings:
  #The default format that worlds will use when created
  default-format: leveldb

**Comment by VixikHD** [comment]
This should be debugged in 1.4
